src/components/Article.jsx

```diff
@@ -32,8 +48,17 @@ function Article() {
         <img src={article.article_img_url} />
         <div className="votes">
           <p>
-            <strong>Votes:</strong> {article.votes}
+            <strong>Votes:</strong> {votes}
+            {error && <span className="error-msg">{error}</span>}
           </p>
+          <div className="voting-buttons">
+            <button type="button" onClick={() => handleVoting(1)}>
+              Up vote
+            </button>
+            <button type="button" onClick={() => handleVoting(-1)}>
+              Down vote
+            </button>
+          </div>
         </div>
         <p>{article.body}</p>
```

Comment:
Something you could consider is do you want a user to be able to vote multiple times up or down or just once?